### PR TITLE
doc: Doc fixes

### PR DESCRIPTION
**Pull request by umapraseeda**
Doc fixes 
Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_radio_coex.rst [hunk 1]
```diff
@@ -220,8 +220,8 @@ To use it, complete the following steps:
 
 1. Add the devicetree `nrf_radio_coex` node as described in :ref:`ug_radio_cx_nrf700x_hw_if`.
 2. If :kconfig:option:`CONFIG_WIFI` is set to ``y`` and one of the supported SR protocols is enabled no additional steps are needed.
-   If you do not wish to use the radio coexistence implementation, you must set  :kconfig:option:`CONFIG_MPSL_CX` to ``n``
-3. If :kconfig:option:`CONFIG_WIFI` is disabled or none of the supported SR protocols are enabled the implementation must be enabled manually with :kconfig:option:`CONFIG_MPSL_CX` and :kconfig:option:`CONFIG_MPSL_CX_NRF700X`
+   If you do not wish to use the radio coexistence implementation, you must set  :kconfig:option:`CONFIG_MPSL_CX` to ``n``.
+3. If :kconfig:option:`CONFIG_WIFI` is disabled or none of the supported SR protocols are enabled the implementation must be enabled manually with :kconfig:option:`CONFIG_MPSL_CX` and :kconfig:option:`CONFIG_MPSL_CX_NRF700X`.
```
Comment (peknis) [inline doc/nrf/ug_radio_coex.rst:224]:
```suggestion
3. If :kconfig:option:`CONFIG_WIFI` is disabled or none of the supported SR protocols are enabled, the implementation must be enabled manually with :kconfig:option:`CONFIG_MPSL_CX` and :kconfig:option:`CONFIG_MPSL_CX_NRF700X`.
```

File: include/net/lwm2m_client_utils.h [hunk 2]
```diff
@@ -62,7 +62,7 @@ struct modem_mode_change {
  * storage using Zephyr settings API. Credential are stored to
  * modem and no keys would enter the flash.
  *
- * @note This API calls settings_subsys_init() so should
+ * @note This API calls settings_subsys_init() and so should
```
Comment (peknis) [inline include/net/lwm2m_client_utils.h:65]:
```suggestion
 * @note This API calls settings_subsys_init() and should
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -241,14 +235,12 @@ Bluetooth mesh samples
 
 * :ref:`bluetooth_mesh_sensor_client` sample:
 
-  * Added:
-
-    * Ability to use buttons to send get and set messages for a sensor setting, as well as a get message for a sensor descriptor.
+  * Added the ability to use buttons to send get and set messages for a sensor setting, as well as a get message for a sensor descriptor.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:238]:
```suggestion
  * Added the ability to use buttons to send ``get`` and ``set`` messages for a sensor setting, as well as a ``get`` message for a sensor descriptor.
```
Not 100% sure about this, but I would prefer some highlighting for the message types to make them stand out from the text.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -335,9 +319,9 @@ NFC samples
 nRF5340 samples
 ---------------
 
-* :ref:`multiprotocol-rpmsg-sample`:
+* :ref:`multiprotocol-rpmsg-sample` sample:
 
-  * Decreased the maximum supported number of concurrent Bluetooth LE connections to 4.
+  * Updated by decreasing the maximum supported number of concurrent Bluetooth LE connections to 4.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:324]: 4 -> four ?

File: doc/nrf/releases/release-notes-changelog.rst [hunk 5]
```diff
@@ -481,10 +472,12 @@ Modem libraries
...
 * :ref:`lte_lc_readme` library:
 
-  * Added support for GCI (Global Cell ID) neighbor cell measurement search types, which are supported by the modem firmware versions >= 1.3.4.
+  * Added support for GCI (Global Cell ID) neighbor cell measurement search types, which are supported by the modem firmware versions from and including v1.3.4.
+
+   * Updated:
 
-    Parameter type in :c:func:`lte_lc_neighbor_cell_measurement` changed to :c:struct:`lte_lc_ncellmeas_params`.
-    It includes both search type and GCI count that have an impact only with GCI search types.
+     * Parameter type in :c:func:`lte_lc_neighbor_cell_measurement` changed to :c:struct:`lte_lc_ncellmeas_params`.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:479]:
```suggestion
     * Parameter type in the :c:func:`lte_lc_neighbor_cell_measurement` function changed to :c:struct:`lte_lc_ncellmeas_params`.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/ug_radio_coex.rst [hunk 6]
```diff
@@ -61,7 +61,7 @@ To enable Wi-Fi Coexistence, do as follows:
    See :ref:`ug_multi_image`.
 
 .. note::
-   When using the :ref:`ug_radio_cx_nrf700x_impl` implementation manually selecting the :kconfig:option:`CONFIG_MPSL_CX` Kconfig option is not needed.
+   When using the :ref:`ug_radio_cx_nrf700x_impl` manually selecting the :kconfig:option:`CONFIG_MPSL_CX` Kconfig option is not needed.
```
[the remark this answers is not in the document]
Comment (umapraseeda) [reply doc/nrf/ug_radio_coex.rst:64]: @FrancescoSer could you check/suggest? Would this be correct: When using the :ref:`ug_radio_cx_nrf700x_impl`, manually selecting the :kconfig:option:`CONFIG_MPSL_CX` Kconfig option is not needed.

File: samples/wifi/shell/README.rst [hunk 7]
```diff
@@ -38,17 +38,23 @@ Currently, the following configurations are supported:
...
+
+.. code-block::console
 
    west build -b nrf5340dk_nrf5340_cpuapp -- -DSHIELD=nrf7002_ek
 
 To build for the nRF9160 DK, use the ``nrf9160dk_nrf9160_ns`` build target with the ``SHIELD`` CMake option set to ``nrf7002_ek`` and scan-only overlay configuration.
-The following is an example of the CLI command::
+The following is an example of the CLI command:
+
+.. code-block::console
```
Comment (FrancescoSer) [inline samples/wifi/shell/README.rst:57]:
```suggestion
.. code-block:: console
```

**Review by FrancescoSer — CHANGES_REQUESTED** [review]

File: doc/nrf/ug_radio_coex.rst [hunk 8]
```diff
@@ -61,7 +61,7 @@ To enable Wi-Fi Coexistence, do as follows:
    See :ref:`ug_multi_image`.
 
 .. note::
-   When using the :ref:`ug_radio_cx_nrf700x_impl` implementation manually selecting the :kconfig:option:`CONFIG_MPSL_CX` Kconfig option is not needed.
+   When using the :ref:`ug_radio_cx_nrf700x_impl`, manually selecting the :kconfig:option:`CONFIG_MPSL_CX` Kconfig option is not needed.
```
Comment (peknis) [inline doc/nrf/ug_radio_coex.rst:64]:
```suggestion
   When using the :ref:`ug_radio_cx_nrf700x_impl`, you do not need to select the :kconfig:option:`CONFIG_MPSL_CX` Kconfig option manually.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by divipillai — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]

**Review by pdunaj — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]

**Comment by umapraseeda** [comment]
The fixes to the changelog will be applied later while creating the release notes from the changelog. Removing that from the PR.

**Review by Damian-Nordic — APPROVED** [review]

**Review by joerchan — CHANGES_REQUESTED** [review]

File: samples/bluetooth/throughput/README.rst [hunk 9]
```diff
@@ -113,10 +113,8 @@ After programming the sample to both kits, complete following steps to test it:
 
 1. |connect_terminal_both_ANSI|
 #. Reset both kits.
-#. Press **Button 1** on the first development kit or type ``central`` in the terminal
-   connected to the first kit to set it into the central (tester) role.
-#. Press **Button 2** on the second development kit or type ``peripheral`` in the terminal
-   connected to the second kit to set it into the peripheral (peer) role.
+#. Press **Button 1** on the first development kit or type ``central`` in the terminal connected to the first kit to set it into the central (tester) role.
+#. Press **Button 2** on the second development kit or type ``peripheral`` in the terminalconnected to the second kit to set it into the peripheral (peer) role.
```
[the remark this answers is not in the document]
Comment (umapraseeda) [reply samples/bluetooth/throughput/README.rst]: fixed

**Review by joerchan — APPROVED** [review]

**Pull request merged**